Gems/Terrain/Assets/Passes/TerrainParentPass.pass

```diff
@@ -6,16 +6,16 @@
         "PassTemplate": {
             "Name": "TerrainParentPassTemplate",
             "PassClass": "ParentPass",
-            "PassRequests": [			
+            "PassRequests": [
                 {
                     "Name": "TerrainDetailTextureComputePass",
                     "TemplateName": "TerrainDetailTextureComputePassTemplate",
-                    "Enabled": false
+                    "Enabled": true
                 },
                 {
-                    "Name": "TerrainMacroTextureComputePass",
-                    "TemplateName": "TerrainMacroTextureComputePassTemplate",
-                    "Enabled": false
+                    "Name": "TerrainMacroTextureClipmapGenerationPass",
+                    "TemplateName": "TerrainMacroTextureClipmapGenerationPassTemplate",
+                    "Enabled": true
```

Comment:
Do you mean to enable these passes? it seems like you'd want to keep them disabled by default until everything's ready.